.github/workflows/tests.yml

```diff
@@ -96,43 +110,74 @@ jobs:
           composer-options: ${{ matrix.php == '8.0' && '--ignore-platform-reqs' || '' }}
 
       - name: Run tests
-        if: ${{ matrix.php != '8.1' || matrix.redis != '7.4' }}
+        if: ${{ matrix.php != '8.4' || matrix.redis != '7.4' }}
         run: vendor/bin/phpunit
 
       - name: Run tests with coverage
-        if: ${{ matrix.php == '8.1' && matrix.redis == '7.4' }}
-        run: vendor/bin/phpunit --coverage-clover build/logs/clover-default.xml --coverage-filter ./src
+        if: ${{ matrix.php == '8.4' && matrix.redis == '7.4' }}
+        run: vendor/bin/phpunit --coverage-php build/cov/coverage-predis.cov --coverage-filter ./src
 
       - name: Run tests using Relay
         if: ${{ matrix.php != '8.4' && matrix.redis >= '6.2' }}
         run: vendor/bin/phpunit -c phpunit.relay.xml
 
       - name: Run tests using Relay with coverage
-        if: ${{ matrix.php == '8.1' && matrix.redis == '7.4' }}
-        run: vendor/bin/phpunit -c phpunit.relay.xml --coverage-clover build/logs/clover-relay.xml --coverage-filter ./src
+        if: ${{ matrix.php == '8.4' && matrix.redis == '7.4' }}
+        run: vendor/bin/phpunit -c phpunit.relay.xml --coverage-php build/cov/coverage-relay.cov --coverage-filter ./src
 
       - name: Run stack tests
-        if: ${{ matrix.redis == '7.4' }}
+        if: ${{ (matrix.php != '8.4' || matrix.redis != '7.4') && matrix.redis >= '6.2' }}
```

Comment:
No versions of Redis below 6.2 are used in the matrix

Reply:
same as above